src/explore-education-statistics-api-docs/source/getting-started/how-to-get-csv-data/index.html.md.erb

```diff
@@ -24,9 +24,14 @@ examples.
 
 ## Get an entire data set as a CSV file
 
-Every data set is created from an underlying CSV file that contains  **all** the data. You may find
-it useful (or necessary) to work with the underlying CSV instead of interacting with the data set
-via the API.
+It should be noted that the CSV data uses human-readable labels for its contents instead of
+machine-readable identifiers. This means that unlike the rest of the API, there are no backward
```

Comment:
I'd suggest a slight reword if that's OK?

```
This means that unlike the rest of the API, there are no backward
compatibility guarantees between different versions of the data set if your downstream processes rely on labels to always remain the same.
```

Reply:
Definitely okay - I prefer your version! Have swapped it in now